runtime/lua/vim/filetype/detect.lua

```diff
@@ -409,4 +407,7 @@ function M.y(bufnr)
   vim.bo[bufnr].filetype = "yacc"
 end
 
+-- luacheck: pop
```

Comment:
Not sure if push / pop is necessary or if
```
-- luacheck: push no unused args
-- luacheck: push ignore 122
```
would be enough.

Reply:
I think `pop` is correct -- we do that in `filetype.lua` as well.

Reply:
Yes its correct but only required when the errors should not be ignored for the whole file (only up to the pop statement).

Reply:
True, but still good practice :)